tests/foreman/cli/test_partitiontable.py

```diff
@@ -96,7 +96,7 @@ def test_positive_create_with_content(self):
         content = 'Fake ptable'
         ptable = make_partition_table({'content': content})
         ptable_content = PartitionTable().dump({'id': ptable['id']})
-        assert content in ptable_content[0]
+        assert content in ptable_content
```

Comment:
was it intentional ? seems like not related to ssh

Reply:
yes, the dump method returns the stdout of the dump subcommand